_includes/layouts/base.njk

```diff
@@ -15,15 +15,15 @@
 				<div class="header-main__nav">
 					{% include "components/nav.njk" %}
 				</div>
-				<div class="header-main__menu js-closed">
-					<div class="header-main__menu__text js-closed-text">Menu</div>
-					<div class="header-main__menu__text js-open-text">Close</div>
+				<button class="header-main__menu js-closed" aria-expanded="false">
+					<div class="header-main__menu__text closed-text">Open menu</div>
```

Comment:
The only thing I would change is to make the additional text screen reader only, then we can keep the text the same as Lyndsay's design for non-screen readers

Reply:
Ah ok, I understand. Re-pushed with aria-label attribute added.